shell/components/SimpleBox.vue

```diff
@@ -31,6 +31,7 @@ export default {
   <div
     v-if="shown"
     class="simple-box"
+    data-testid="simple-box-container"
```

Comment:
comment only - we might run into issues when there's more than one of these on screen, but at the moment it's fine